lib/Sema/CSApply.cpp

```diff
@@ -4349,7 +4356,8 @@ namespace {
       // If we weren't explicitly told by the caller which disjunction choice,
       // get it from the solution to determine whether we've picked a coercion
       // or a bridging conversion.
-      auto *locator = cs.getConstraintLocator(expr);
+      auto *locator =
+          cs.getConstraintLocator(expr, LocatorPathElt::CoercionOperand());
```

Comment:
Nit: you can use `ConstraintLocator::CoercionOperand` if path element doesn't have any parameters.